TheKoans/AboutStrings.cs

```diff
@@ -274,15 +274,15 @@ public void UnderstandingThePerformanceOfCheckingForEmptyStrings()
 
             // Maybe there's a better way of sharing this information, but just add true
             // below after reading this.
-            Assert.AreEqual(true, FILL_ME_IN, "Sometimes you need to take time and stop to smell the performance..");
+            Assert.AreEqual(true, true, "Sometimes you need to take time and stop to smell the performance..");
         }
 
         [TestMethod]
         public void CaseMattersWhenComparingStrings()
         {
             var strProper = "United States of America";
             var strUpper = "UNITED STATES OF AMERICA";
-            Assert.AreEqual(FILL_ME_IN, strProper.Equals(strUpper), "It's not what you say, it's how you say it.");
+            Assert.AreEqual(false, strProper.Equals(strUpper), "It's not what you say, it's how you say it.");
```

Comment:
Good job.  Another thing you could do is to say Assert.AreEqual(true,strProper.ToUpper().Equals(strProper));